app/Http/Requests/PositionFormRequest.php

```diff
@@ -21,10 +23,43 @@ public function authorize()
      *
      * @return array
      */
-    public function rules()
+    public static function rulesStore($request)
     {
-        return [
-            'name' => 'required|min:2|max:255|unique:positions,name,' . $this->id,
+        $messages = [
+            'name.required' => __('Position') . __('Required'),
+            'name.string' => __('Position') . __('String'),
+            'name.unique' => __('Position') . __('Unique'),
         ];
+
+        return Validator::make($request->all(),
+            [
+                'name' => 'required|string|unique:positions',
+            ], $messages
+        );
+    }
+
+    public static function rulesUpdate($request, $id)
+    {
+        $messages = [
+            'name.required' => __('Position') . __('Required'),
+            'name.string' => __('Position') . __('String'),
+            'name.unique' => __('Position') . __('Unique'),
+        ];
+
+        $position = Position::findOrFail($id);
+
+        if ($position->name == $request->name) {
+            return Validator::make($request->all(),
+                [
+                    'name' => 'required|string',
+                ], $messages
+            );
+        } elseif ($position->name != $request->name) {
+            return Validator::make($request->all(),
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.ContentAfterOpenBracket</br>**Severity**: error</br>**File**: app/Http/Requests/PositionFormRequest.php</br>**Line**: 58</br>Opening parenthesis of a multi-line function call must be the last content on the line